action.yml

```diff
@@ -1,8 +1,12 @@
-name: 'Wait'
+name: 'Constable'
 description: 'Create a JavaScript Action grade the project'
+inputs:
+
 outputs:
-  grade:
-    description: 'The output of this file will be a grade and percentage given to the repo.'
+  score:
+    description: 'The repository is assigned a score based on the evaluation criterias'
+  feedback:
+    description: 'Feedback on how to improve the repo is provided'
 runs:
   using: 'node12'
-  main: 'dist/index.js'
+  main: 'main/index.js'
```

Comment:
? where is the `main` directory?

We should be packaging the index.js file via ncc. The command for that is `npm run package` and then GitHub uses that file for running. So this should stay as `dist/index.js`

Reply:
My bad. main folder came with the action.yml file before I started editing it.